platform/android/java/build.gradle

```diff
@@ -158,6 +158,8 @@ task zipCustomBuild(type: Zip) {
     }
     from(fileTree(dir: 'app', excludes: ['**/build/**', '**/.gradle/**', '**/*.iml']), fileTree(dir: '.', includes: ['gradlew', 'gradlew.bat', 'gradle/**']))
     include '**/*'
+    from(fileTree(dir: 'lib', includes: ['**/libs/**']))
+    include 'openxr/**/*'
```

Comment:
@kisg Do we need to include this given that it's not being used by the custom build?

Reply:
No, if we agree to provide the Khronos loader as a separate AAR plugin, then we can remove this. Will do it in an update.

Reply:
I think we pretty much agreed on that so ... :P

Reply:
Done.